data/binding/binditems.go

```diff
@@ -0,0 +1,244 @@
+// auto-generated
+// **** THIS FILE IS AUTO-GENERATED, PLEASE DO NOT EDIT IT **** //
+
+package binding
+
+// Bool supports binding a bool value in a Fyne application
+type Bool interface {
+	DataItem
+	Get() bool
+	Set(bool)
+}
+
+// NewBool returns a bindable bool value that is managed internally.
+func NewBool() Bool {
+	blank := false
+	return &bindBool{val: &blank}
+}
+
+// BindBool returns a new bindable value that controls the contents of the provided bool variable.
+func BindBool(v *bool) Bool {
+	if v == nil {
+		return NewBool() // never allow a nil value pointer
+	}
+
+	return &bindBool{val: v}
+}
+
+type bindBool struct {
```

Comment:
Consider a more noun-ish naming convention;
```suggestion
type boundBool struct {
```

Reply:
Thanks, good catch.